src/NodaTime.Test/OffsetDateTest.cs

```diff
@@ -148,5 +151,39 @@ public void Deconstruction()
                 Assert.AreEqual(offset, actualOffset);
             });
         }
+
+        [Test]
+        public void BinarySerialization()
+        {
+            var date = new LocalDate(2013, 4, 12);
+            var offset = Offset.FromHoursAndMinutes(5, 30);
+            var value = new OffsetDate(date, offset);
+            TestHelper.AssertBinaryRoundtrip(value);
+        }
+
+        [Test]
+        public void XmlSerialization_Iso()
```

Comment:
Also a test that clarifies whether 123 AD is `123-01-01Z` or (what I believe might be required for `xs:date`) `0123-01-01Z` would be nice.

Reply:
It's 0123-01-01 - all our default formatters use `uuuu` as the year specifier.